final_prep/annotations.rb

```diff
@@ -3,41 +3,53 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+# defines the method build_a_bear with five arguements
 def build_a_bear(name, age, fur, clothes, special_power)
+  # defines method variables using argument input
+  # defines interpolated string
   greeting = "Hey partner! My name is #{name} - will you be my friend?!"
+  # defines variable with array of argument data
   demographics = [name, age]
+  # defines interpolated string
   power_saying = "Did you know that I can #{special_power}?"
+  # defines a hash for a built_bear and continues to use arguments given by the above method
   built_bear = {
     'basic_info' => demographics,
```

Comment:
With very complex data, you may want to track whether variable values come from arguments, or somewhere else, for example, the variables like `demographics` defined above.